lib/primo/search.rb

```diff
@@ -51,7 +51,22 @@ def self.get(params = {})
       params = params.to_h.transform_keys(&:to_sym)
       method = get_method params
       (url, query) = url(params)
-      new super(url, query:, timeout: Primo.configuration.timeout(params)), method
+
+      unless Primo.configuration.enable_retries
```

Comment:
This unless is redundant.  You already have logic for when a retry should happen below.